Tests/SwiftParserTest/Types.swift

```diff
@@ -355,12 +355,89 @@ final class TypeParameterPackTests: XCTestCase {
       """)
   }
   func testParameterPacks28() throws {
-    // We allow whitespace between the generic parameter and the '...', this is
-    // consistent with regular variadic parameters.
+      // We allow whitespace between the generic parameter and the '...', this is
+      // consistent with regular variadic parameters.
+      AssertParse(
+    """
+    func f1<T ...>(_ x: T ...) -> (T ...) {}
+    """)
```

Comment:
Indentation seems off here.

Reply:
Indeed, also the indention below. I changed it.